packages/design-tokens/tokens/color/background.json

```diff
@@ -0,0 +1,39 @@
+{
+  "color": {
+    "background": {
+      "base": { "value": "{color.base.white.value}" },
+      "alt": { "value": "{color.core.neutral.1.value}" },
+      "inverse": { "value": "{color.core.neutral.6.value}" },
+      "success": { "value": "{color.base.green.value}" },
+      "error": { "value": "{color.base.red.value}" },
+      "warning": { "value": "{color.base.orange.value}" },
+      "info": { "value": "{color.base.blue.value}" },
+
+      "button": {
+        "primary": {
+          "base": { "value": "{color.base.green.value}" },
+          "hover": { "value": "{color.base.green.value}" },
```

Comment:
We do a lot in the css right now to adjust the color:
```scss
    &:hover {
      background-color: $crl-green-4;
      color: $crl-base-white;
    }

    &:active {
      background-color: $crl-green-4;
      border-width: 1px;
      border-color: mix(black, $crl-green-4, 10%);
    }

    &:focus {
      border-width: 1px;
      border-color: mix(black, $crl-green-4, 20%);
      @include crl-box-shadow(0, 0, 0, 3px, rgba($crl-base-green, 0.4));
    }

    &:disabled {
      background-color: $crl-green-2;
      border-color: $crl-green-2;

      &:hover {
        cursor: not-allowed;
      }
    }
```

Reply:
I wonder if we could add an additional layer to the states with separate colors. For example for `hover`,
```json
...
"hover": {
  "background": { "value": "{color.base.green.value}" },
  "foreground": { "value": "{color.base.white.value}" },
},
```

I think should also move ahead and iterate. There may be some way to capture what Rachel had mentioned this morning about a number value representing a percent _darker_ or whathaveyou.

Reply:
Right now the text color is separate from the background. For example, the button text color on hover is [here](https://github.com/cockroachdb/ui/pull/54/files#diff-55f5b9d8b713974d4b9cc2eb1deb9af0R43). Is that what you mean by foreground?